rust/db_handling/src/identities.rs

```diff
@@ -100,6 +100,40 @@ lazy_static! {
     static ref REG_PATH: Regex = Regex::new(r"^(?P<path>(//?[^/]+)*)(///(?P<password>.+))?$").expect("known value");
 }
 
+#[derive(Clone, Encode, Decode)]
+pub enum SignaturesBulk {
+    #[codec(index = 4)]
+    V1(SignaturesBulkV1),
+}
+
+impl From<SignaturesBulkV1> for SignaturesBulk {
+    fn from(s: SignaturesBulkV1) -> Self {
+        Self::V1(s)
+    }
+}
+
+#[derive(Clone, Encode, Decode)]
+pub struct SignaturesBulkV1 {
+    /// `MultiSignature`s in encoded form.
+    signatures: Vec<Vec<u8>>,
```

Comment:
@pgolovkin I am going to change this to typed signatures: `Vec<MultiSignature>` if that is ok with you